clmm/support/mock_data.py

```diff
@@ -118,8 +118,9 @@ def generate_galaxy_catalog(cluster_m, cluster_z, cluster_c, cosmo, zsrc,
         table will not include this column.
     photoz_sigma_unscaled : float, optional
         If set, applies photo-z errors to source redshifts
-    pz_bin_width: float
-        Width of photo-z bins
+    pz_bins: int, array
+        Pz pdf bins in the given range. If int, the limits are set automatically.
```

Comment:
If we want the docstrings to be consistent here, maybe consider change Pz pdf to photo-z pdf.